internal/bminventory/inventory.go

```diff
@@ -5866,3 +5878,55 @@ func isBaremetalBinaryFromAnotherReleaseImageRequired(cpuArchitecture, version s
 		featuresupport.IsFeatureSupported(version,
 			models.FeatureSupportLevelFeaturesItems0FeatureIDARM64ARCHITECTUREWITHCLUSTERMANAGEDNETWORKING)
 }
+
+// updateMonitoredOperators checks the content of the installer configuration and updates the list
+// of monitored operators accordingly. For example, if the installer configuration uses the
+// capabilities mechanism to disable the console then the console operator is removed from the list
+// of monitored operators.
+func (b *bareMetalInventory) updateMonitoredOperators(tx *gorm.DB, cluster *common.Cluster) error {
+	// Get the complete installer configuration, including the overrides:
+	installConfigData, err := b.installConfigBuilder.GetInstallConfig(cluster, false, "")
+	if err != nil {
+		return err
+	}
+	var installConfig installcfgdata.InstallerConfigBaremetal
+	err = yaml.Unmarshal(installConfigData, &installConfig)
+	if err != nil {
+		return err
+	}
+
+	// Since version 4.11 it is possible to disable the console via the capabilities section of
```

Comment:
Guessing we currently support this feature with installConfigOverrides

Reply:
Yes, the user has to explicitly disable the console with something like this:

```yaml
capabilities:
  baselineCapabilitySet: None
  additionalEnabledCapabilities:
  - baremetal
```